jwst-masterclass/0.1.0/Dockerfile

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Association of Universities for Research in Astronomy
+# Distributed under the terms of the Modified BSD License.
+
+FROM jupyter/scipy-notebook:a238993ad594
+# See https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html
+# for a list of what comes with the base image
+# python version is: 3.6.8
+
+LABEL maintainer="Science Platforms <cbrasseur@stsci.edu>"
+
+# Environment variables
+ENV MKL_THREADING_LAYER="GNU"
+
+# CRDS environment variables
+ENV CRDS_PATH=/home/jovyan/crds_cache
+ENV CRDS_SERVER_URL=https://jwst-serverless.stsci.edu
+ENV CRDS_S3_ENABLED=1
+ENV CRDS_S3_RETURN_URI=0
+ENV CRDS_MAPPING_URI=s3://dmd-test-crds/mappings/jwst
+ENV CRDS_REFERENCE_URI=s3://dmd-test-crds/references/jwst
+ENV CRDS_CONFIG_URI=s3://dmd-test-crds/config/jwst
+ENV CRDS_USE_PICKLES=0
+ENV CRDS_DOWNLOAD_MODE=plugin
+ENV CRDS_DOWNLOAD_PLUGIN='crds_s3_get ${SOURCE_URL} ${OUTPUT_PATH} ${FILE_SIZE} ${FILE_SHA1SUM}'
+ENV CRDS_CONTEXT=jwst_0535.pmap
```

Comment:
We shouldn't set this variable in this case, we're using jwst master here so we can also use the bleeding edge context.

Reply:
Ah, I see.

For my own education, what is a pmap file?

Reply:
That filename represents a CRDS context, which is a specific state of reference files and rules files that CRDS uses when we ask it to make recommendations.  It's sort of the "commit hash" of the "repo" of CRDS data.

The file itself contains Python code that describes the relevant files, but here we're just using it to identify a context.

Reply:
@cslocum this still needs to change I think?